webapp/app/js/filters.js

```diff
@@ -40,22 +40,39 @@ treeherder.filter('stripHtml', function() {
 treeherder.filter('linkifyBugs', function() {
     return function(input) {
         var str = input || '';
-        var bug_matches = /.*-- ([0-9]+)|.*bug-([0-9]+)|.*Bug ([0-9]+)/ig.exec(str);
-        var pr_matches = /PR#([0-9]+)/i.exec(str);
         var clear_attr = 'ignore-job-clear-on-click';
-        if (pr_matches) {
-            var pr_url = "https://github.com/mozilla-b2g/gaia/pull/" + pr_matches[1];
-            var pr_hyperlink = '<a href="' + pr_url + '" ' + clear_attr + '>' +
-                               pr_matches[1] + '</a>';
-            str = str.replace(pr_matches[1], pr_hyperlink);
-        }
+
+        // Test strings
+        // var str = "No bugs";
+        // var str = "One bug 1130488";
+        // var str = "One PR#456";
+        // var str = "Bug 1130488 disable tests for bug 999 and -- 777 and PR#368 and PR#456";
```

Comment:
You can take these comments out, there's no need for them in production code as long as it works. :)